nbbuild/antsrc/org/netbeans/nbbuild/extlibs/DownloadBinaries.java

```diff
@@ -80,6 +80,19 @@ public void setServer(String server) {
         this.server = server;
     }
 
+    private String repos = "https://repo1.maven.org/maven2/";
```

Comment:
This feels a bit sneaked in (support for non-central repositories), but I see why it might be a good idea.

Reply:
Yes, Gradle do not publish to Maven Central and I realized just after I've implemented this feature that we did have an issue using non-maven central repositories. I've ended up uploading the Gradle Tooling API to OSUL, but I kept the feature in.